jenkinshelper.ksh

```diff
@@ -0,0 +1,139 @@
+#!/bin/ksh
```

Comment:
Did you consider to move this file to the tools folder?

Reply:
Ah, no forgot bout that. Moving it there. The Jenkinsfile needs to be in the repo root though.